source/install-riallto-linux.rst

```diff
@@ -0,0 +1,59 @@
+.. _install-riallto-Linux:
+
+Install Riallto on Linux 
+===========================
```

Comment:
Length should be equal as heading?
You can also use Markdown if it is easier

Reply:
Ah great, yeah I'll just switch to markdown.